precompile/wasm/IWasm.sol

```diff
@@ -32,20 +32,21 @@ interface IWasm {
     function executeContract(
         address sender,
         address contractAddress,
-        bytes calldata msg,
+        bytes calldata message,
         string calldata denom,
         uint256 amount
     ) external returns (bytes calldata data);
     function migrateContract(
         address sender,
         address contractAddress,
         uint256 codeId,
-        bytes calldata msg
+        bytes calldata message
     ) external returns (bytes calldata data);
     
     // Queries
     function smartContractState(
         address contractAddress,
         bytes calldata queryData
     ) external view returns (bytes calldata data);
+    function associatedAddress(address evmAddress) external view returns (bytes calldata addr);
```

Comment:
This seems like a very good access point!
However, the current precompiled contract is written based on msg-server or query endpoint. Since it is a function(associatedAddress) that does not exist in wasmd, it seems the right direction is to write a precompiled contract for the `auth` module and use `GetAccount` in the contract.

Reply:
[d5836fd](https://github.com/xpladev/xpla/pull/161/commits/d5836fdf2e82e087a89763de125eb7eb0e9d804f)

Reply:
I think it would be better to implement it as `GetAccount` rather than `associatedAddress`, which is a function that does not exist.

Reply:
[000a72a](https://github.com/xpladev/xpla/pull/161/commits/000a72ad10fb2725d1a99269938c84f1320d29b8)